### ISSUE TITLE
Crash using Analyze Range in Graph Editor

### ISSUE THREAD

**Issue by rcorre** [Zylann/godot_voxel#424]
Godot: 4.0.alpha.custom_build.8243c7ab5
Voxel: 31a7ddbf838572e50415159a56720275f9523262

Same example as another issue: https://github.com/Zylann/godot_voxel/files/9330562/example.zip

Enable "Analyze Range" in the graph editor, it crashes trying to access a null CanvasItem:

```
(gdb) bt
#0  0x00007f90e428e36c in ?? () from /usr/lib/libc.so.6
#1  0x00007f90e423e838 in raise () from /usr/lib/libc.so.6
#2  0x00007f90e4228535 in abort () from /usr/lib/libc.so.6
#3  0x000055a984771d1f in handle_crash (sig=<optimized out>) at platform/linuxbsd/crash_handler_linuxbsd.cpp:131
#4  <signal handler called>
#5  0x000055a986c00e40 in Color::operator== (p_color=..., this=0x410) at ./core/math/color.h:72
#6  CanvasItem::set_modulate (this=0x0, p_modulate=...) at scene/main/canvas_item.cpp:372
#7  0x000055a984c9bb6e in zylann::voxel::VoxelGraphEditor::update_range_analysis_previews (this=0x55a99d587af0) at ./core/math/color.h:245
#8  0x000055a984c9fc86 in zylann::voxel::VoxelGraphEditor::update_previews (this=0x55a99d587af0, with_live_update=<optimized out>) at modules/voxel/editor/graph/voxel_graph_editor.cpp:668
#9  0x000055a984ca030a in zylann::voxel::VoxelGraphEditor::_process (this=0x55a99d587af0, delta=<optimized out>) at modules/voxel/editor/graph/voxel_graph_editor.cpp:221
#10 0x000055a9889c54a4 in Object::notification (this=0x55a99d587af0, p_notification=25, p_reversed=<optimized out>) at core/object/object.cpp:737
#11 0x000055a986c90d3e in SceneTree::_notify_group_pause (this=0x55a990fab0b0, p_group=..., p_notification=25) at scene/main/scene_tree.cpp:904
#12 0x000055a986c92f9f in SceneTree::process (this=0x55a990fab0b0, p_time=0.0069009999999999991) at scene/main/scene_tree.cpp:453
#13 0x000055a9847add33 in Main::iteration () at main/main.cpp:2884
#14 0x000055a9847722b9 in OS_LinuxBSD::run (this=this@entry=0x7ffe15fd5560) at platform/linuxbsd/os_linuxbsd.cpp:537
#15 0x000055a984763e93 in main (argc=<optimized out>, argv=0x7ffe15fd5a98) at platform/linuxbsd/godot_linuxbsd.cpp:72
```

**Comment by Zylann** [comment]
Should be fixed by 5c4718877362eccb3d066f81cef943970cdf8cb7

**Comment by rcorre** [comment]
Fixed, thanks!
